cnf-tests/Dockerfile

```diff
@@ -61,6 +59,9 @@ RUN yum install -y numactl-devel make gcc && \
     cp hwlatdetect /hwlatdetect && \
     cp cyclictest /cyclictest
 
+FROM quay.io/openshift/origin-oc-rpms:4.13 AS oc
+
+# Final image
 FROM centos:7
```

Comment:
for a separate PR we must update this image

Reply:
will do